cpp/StorageClientFactory.cpp

```diff
@@ -27,13 +27,8 @@ IStorageClient * StorageClientFactory::getClient(
     case StageType::MOCKED_STAGE_TYPE:
       return injectedClient;
     case StageType::AZURE:
-#ifdef __linux__
       CXX_LOG_INFO("Creating Azure client");
       return new SnowflakeAzureClient(stageInfo, parallel, transferConfig);
```

Comment:
I think we should hold off until we clear security review. When merging lets throw Exception saying Azure Put Get is not supported on ODBC.